### ISSUE TITLE
v4 beta 8 - Logging out does not delete session

### ISSUE THREAD

**Issue by mluttmann** [auth0/nextjs-auth0#1826]
### Checklist

- [X] The issue can be reproduced in the [nextjs-auth0 sample app](https://github.com/auth0-samples/auth0-nextjs-samples/tree/main/Sample-01) (or N/A).
- [X] I have looked into the [Readme](https://github.com/auth0/nextjs-auth0#readme), [Examples](https://github.com/auth0/nextjs-auth0/blob/main/EXAMPLES.md), and [FAQ](https://github.com/auth0/nextjs-auth0/blob/main/FAQ.md) and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/nextjs-auth0/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/nextjs-auth0/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

When redirecting the user to `/auth/logout`, the session is not deleted and the user is still logged in. The server logs `The Auth0 client does not have RP-initiated logout enabled, the user will be redirected to the '/v2/logout' endpoint instead. Learn how to enable it here: https://auth0.com/docs/authenticate/login/logout/log-users-out-of-auth0#enable-endpoint-discovery`. It appears that the RP logout option is part of the Enterprise tier, but is unlikely to be related to the problem described.

### Reproduction

1. Create Next.js app (answer everything with yes)
```
npx create-next-app@latest test-app
cd test-app
```

2. Setup auth0 v4 beta 8 as describe in [v4.0.0-beta.8/README.md](https://github.com/auth0/nextjs-auth0/blob/v4.0.0-beta.8/README.md#getting-started)

3. Add button for logout in `test-app/src/app/page.tsx`
```
    <main>
      <h1>Welcome, {session.user.name}!</h1>
      <a href="/auth/logout">Log out</a>
    </main>
```

4. Start application, log in and try to log out

### nextjs-auth0 version

4.0.0-beta.8

### Next.js version

15.0.3

### Node.js version

22.11.0

**Comment by guabu** [comment]
Hi @mluttmann 👋 Thanks for reporting this — it looks like the cookie is not being cleared for the `/v2/logout` redirect. We'll get a fix for this in the upcoming release!

For the time being, you can enable RP-initiated logout which is available to all customers (not only for paid plans). The link provided in the log provides the steps on how to enable it: https://auth0.com/docs/authenticate/login/logout/log-users-out-of-auth0#enable-endpoint-discovery

**Comment by nandan-bhat** [comment]
Closing this ticket as this issue has been fixed `4.0.0-beta.9`
